### PR TITLE
Improved docstring of ReferenceFrame.orient()

### PR DESCRIPTION
**Pull request by moorepants**
- Clarified many details of the orient() method.
- Improved orientnew() to match orient()
- PEP8 orient() method's code.

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by moorepants** [comment]
Here is the rendered version:

Selection_598
Selection_599
Selection_600
Selection_601
Selection_602

**Comment by moorepants** [comment]
New docstring for orientnew():
Selection_603
Selection_604

**Comment by moorepants** [comment]
This is ready for review.

@sympy/mechanics

**Comment by moorepants** [comment]
Does the coverage thing even work. It says this will decrease coverage by 28%!

**Comment by czgdp1807** [comment]
> Does the coverage thing even work.

I am facing the same issue. The results are incorrect many times. Probably codecov has bugs.

File: sympy/physics/vector/frame.py [hunk 1]
```diff
@@ -420,72 +421,211 @@ def dcm(self, otherframe):
...
+        >>> A = ReferenceFrame('A')
+        >>> A.orient(N, 'DCM', N.dcm(B))
+        >>> A.dcm(N)
+        Matrix([
+        [1,       0,      0],
+        [0,  cos(q1), sin(q1)],
+        [0, -sin(q1), cos(q1)]])
+
+        **Note carefully that ``N.dcm(B)`` was passed into ``orient()`` for
+        ``A.dcm(N)`` to match ``B.dcm(N)``.**
```
Comment (sylee957) [inline sympy/physics/vector/frame.py]: I don't think that emphasis can be used with inline literals.

image

Comment (moorepants) [reply sympy/physics/vector/frame.py]: Fixed, thanks.

**Comment by moorepants** [comment]
I think that this is ready to go in. Can I get an approval from a reviewer?

**Review by sylee957 — APPROVED** [review]

**Comment by moorepants** [comment]
@sylee957 Thanks!

**Pull request merged**